cmd/influxd/main.go

```diff
@@ -61,7 +62,7 @@ type Main struct {
 func NewMain() *Main {
 	return &Main{
 		Logger: zap.New(
-			zap.NewTextEncoder(),
+			zaplogfmt.NewEncoder(),
```

Comment:
What about all the other places we're currently using `zap.NewTextEncoder()`? Do we need to replace those too?